src/app/notification/notification.component.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import { Component, OnInit } from "@angular/core";
+
+import { Broadcaster } from "../shared";
+import { ActionService } from "../room";
+import { ConfigService } from "../config.provider";
+
+@Component({
+  selector: "jh-notification",
+  template: require("./notification.component.html")
+})
+export class NotificationComponent implements OnInit {
```

Comment:
Looks like this class is only used to tell the user he is muted. In that case, `NotificationComponent` looks a little bit generic as a name. Shouldn't be something like `MuteNotificationComponent' (or `MuteAlertComponent` if you want something shorter).